.github/workflows/ci.yml

```diff
@@ -44,3 +44,29 @@ jobs:
 
       - name: Test coverage
         run: npx truffle run coverage
+
+      - name: Upload coverage
+        run: bash <(curl -s https://codecov.io/bash) || true
```

Comment:
You should be able to use the github action instead, right?

https://github.com/marketplace/actions/codecov

Reply:
The bash solution is also officially endorsed by them, i will still take a look.